replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorRunner.java

```diff
@@ -110,8 +110,9 @@ void scheduleReplication() {
             .collect(Collectors.toList());
     try {
       for (ReplicatorConfig config : configsToSchedule) {
-        replicator.submitSyncRequest(
-            new SyncRequestImpl(config, new ReplicationStatusImpl(config.getName())));
+        ReplicationStatusImpl status = new ReplicationStatusImpl();
```

Comment:
I personally preferred the simpleness of the previous code.
I like that we are correlating with config ids and not arbitrary names now.

Reply:
I find it a little easier to read this way and I miss the id constructor a little bit but this aligns more with the way that we initialize our other data objects.